string object_id          # Unique identifier for the object
string object_type   # Type of object (cube, cylinder, sphere)
geometry_msgs/Pose pose   # Position and orientation of the object